QueryProfiler/Profile/ProfileScheme.cs

```diff
@@ -6,19 +6,19 @@
 namespace QueryProfiler
 {
    public class ProfileScheme
-    {     
+    {     // Operators
         public int SearchCounter { get; set; }
         public int ExtendCounter { get; set; }
         public int JoinCounter { get; set; }
         public int LookupCounter { get; set; }
         public int ProjectCounter { get; set; }
         public int ProjectAwayCounter { get; set; }
-        public int InCounter { get; set; }
         public int ProjectKeepCounter { get; set; }

```

Comment:
All of those are more relevant to the improvements the code will suggest
but profile wise, those doesnt carry a major performance effect, so i wouldnt state them as part of the profile returned to user

same for project and project-away